pull.go

```diff
@@ -24,23 +24,28 @@ func (cmd *PullCommand) Run() error {
 		Debug = true
 	}
 
-	client, err := phraseapp.NewClient(cmd.Config.Credentials)
-	if err != nil {
-		return err
-	}
-
-	targets, err := TargetsFromConfig(cmd)
-	if err != nil {
-		return err
-	}
+	err := func() error {
```

Comment:
Good catch, did not remove it. We only reported exceptions for push and pull and reported the errors generically with this error instead of typing ReportError on all occasions. (removed now!)